code/modules/mob/living/carbon/human/ego_gifts.dm

```diff
@@ -1089,6 +1089,14 @@
 	slot = HELMET
 	var/ear_overlay
 
+/datum/ego_gifts/sunshower
+	name = "Sunshower"
+	icon_state = "sunshower"
+	temperance_bonus = 10
```

Comment:
Not necessarily a code concern but more of a balance concern as this is a pretty big temperance creep. There are already issues with players potentially reaching 100% work success rates.

Reply:
Also, I don't see an actual icon for this gift.